docs/copymode.md

```diff
@@ -52,6 +52,7 @@ reassignable.
 |                       | `alt-b`         |
 |                       | `Shift-Tab`     |
 |                       | `b`             |
+| Move forward one word end    | `e` |
```

Comment:
Please rebase and amend this to match the new style used in this doc:

```suggestion
| Move forward one word end    | <kbd>e</kbd> |
```

Reply:
thanks @wez!